eZ/Publish/API/Repository/Values/Content/Field.php

```diff
@@ -76,7 +76,7 @@ public function getValue()
         return $this->value;
     }
 
-    public function getLanguageCode(): string
+    public function getLanguageCode(): ?string
```

Comment:
This is a BC break. You cannot change promise of strict language code into a "maybe a code, maybe a null". What is the actual issue here? Is language code not set?

Reply:
Well, apparently it can be `null`: 
https://github.com/ezsystems/ezplatform-kernel/blob/6c5d6e44399c073e2954baf8e687cd70721c8d63/eZ/Publish/Core/Repository/Tests/Service/Mock/ContentTest.php#L3863
Are data providers invalid then?

There is really no issue besides failing tests, looking at data providers it sure can be `null`.

Reply:
@alongosz thank you Andrzej for the analysis. Of course, I agree with the fact that API changes should be carefully treated, but PR is always a good start to a discussion 💟

Reply:
PHPStan might indeed detect errors in dependent packages after this change, so everyone should be mindful of the fact that this is incoming.